### DISCUSSION TITLE
Globally disable vertical padding on Grid columns?

### DISCUSSION THREAD

**Discussion by disillusionedowl** [mantinedev/mantine#4057] [Q&A]
Is there any other way to do this apart from setting `pt` and `pb` to `0` in `defaultProps` in `theme`? By the way, I would expect this to be off by default, like in Twitter Bootstrap etc. Or maybe rewording `gutter` prop as `gutterH` and adding also `gutterV`?

**Comment by disillusionedowl** [comment]
My current approach, for posterity:

```js
export const theme = {
	components: {
		Grid: {
			defaultProps: {
				mt: 0,
				mb: 0
			}
		},
		GridCol: {
			defaultProps: {
				pt: 0,
				pb: 0
			}
		}
	}
};
```

**Reply by rtivital** [reply]
So, why aren't you satisfied with your current approach?

**Reply by disillusionedowl** [reply]
@rtivital 

I wouldn't say I'm not satisfied, it's great there is a way to do the above and passes by me as a relatively elegant approach for my use case. I don't expect everything to be to my liking by default. I chose Mantine because there were already the most things I liked by default, which is far more than I could ask for.

Having said that, I do have an opinion about the default Grid in Mantine, which should answer your question:

1. Assuming vertical gutter should be the same size as horizontal gutter does not go in line with majority of use cases of any grid system. It's very rare they would need to be the same size in an actual project (not a rapid prototype).

2. There is no API for setting horizontal gutter and vertical gutter separately (e.g. `gutterH` and `gutterV`).

You could argue these are personal preferences but I wouldn't call them that. My personal preference would be having no option for vertical gutter at all, let that be handled by the elements inside the columns. But I can see a lot of people would want to set vertical gutter for some reason.
